### PR TITLE
LOTUS pool

### PR DESCRIPTION
**Pull request by InderKumarRathore**
LOTUS stake pool is single pool and will remain single pool. You can find goals of LOTUS pool here: https://cryptolotus.github.io/lotus/

### CODE REVIEW

File: registry.json [hunk 1]
```diff
@@ -951,5 +951,20 @@
...
+      "facebook": "",
+      "twitter": "https://twitter.com/lotus_ada",
+      "website": "https://cryptolotus.github.io/lotus/",
+      "reddit": "",
+      "instagram": "",
+      "youtube": ""
+    },
+    "telegram": "lotus_ada_pool",
+    "operatorName": "Inder",
+    "github": "cryptolotus"
```
Comment (InderKumarRathore) [inline registry.json:968]: @leo42 can you please merge this PR?

File: registry.json [hunk 2]
```diff
@@ -951,5 +951,20 @@
     "telegram": "edm0nd",
     "operatorName": "Edmond",
     "github": "spmedia"
+  },
+{
+    "ticker": "LOTUS",
+    "poolId": "0126c482283df9e693f28a25fb7e0ad890db5a269c695361069dfc81",
```
Comment (leo42) [inline registry.json:957]: I cannot find your pool.

File: registry.json [hunk 3]
```diff
@@ -951,5 +951,20 @@
...
+    "social": {
+      "facebook": "",
+      "twitter": "https://twitter.com/lotus_ada",
+      "website": "https://cryptolotus.github.io/lotus/",
+      "reddit": "",
+      "instagram": "",
+      "youtube": ""
+    },
+    "telegram": "lotus_ada_pool",
```
Comment (leo42) [inline registry.json:966]: Please use a personal telegram handle!

**Review by leo42 — CHANGES_REQUESTED** [review]

File: registry.json (hunk 2 above)
Comment (InderKumarRathore) [reply registry.json:957]: I fucked ip something 
Retired it will create again

File: registry.json (hunk 1 above)
Comment (leo42) [reply registry.json:968]: @InderKumarRathore  we need a personal Telegram handle. This is to verify your vote when we need to make decisions as a group.

**Review by leo42 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**